laghos_rom.hpp

```diff
@@ -214,6 +245,19 @@ class ROM_Sampler
 
     void Finalize(const double t, const double dt, Vector const& S, Array<int> &cutoff);
 
+    CAROM::StaticSVDOptions GenerateStaticSVDOptions(int dim, int samples_per_time_interval)
+    {
+      CAROM::StaticSVDOptions static_svd_options(dim, samples_per_time_interval);
+      return static_svd_options;
+    }
+
+    CAROM::IncrementalSVDOptions GenerateIncrementalSVDOptions(int dim, double linearity_tol, bool fast_update, int max_basis_dimension, double initial_dt, int samples_per_time_interval, double sampling_tol, double max_time_between_samples)
+    {
+      CAROM::IncrementalSVDOptions inc_svd_options(dim, linearity_tol, max_basis_dimension, initial_dt, samples_per_time_interval, sampling_tol, max_time_between_samples);
+      inc_svd_options.fast_update = fast_update;
```

Comment:
This is a bit verbose compared to before, but it seems it's the only way we can get around potential backwards compatibility issues since partial brace initialization isn't allowed in C++11 when the struct has default values.